src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,125 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationServiceImpl service;
+    private static User user1;
+    private static User user2;
+    private static User user3;
+    private static final long DEFAULT_ID = 200L;
+    private static final String DEFAULT_LOGIN = "sasaylalita";
+    private static final String DEFAULT_PASSWORD = "qwerty12345";
+    private static final int DEFAULT_AGE = 28;
+
+    private void userInit(User user) {
+        user.setLogin(DEFAULT_LOGIN);
+        user.setPassword(DEFAULT_PASSWORD);
+        user.setAge(DEFAULT_AGE);
+    }
+
+    @BeforeAll
+    static void beforeAll() {
+        service = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        user1 = new User();
+        userInit(user1);
+        user2 = new User();
+        userInit(user2);
+        user3 = new User();
+        userInit(user3);
```

Comment:
why do we setup all 3 users for every test? one user is enough. 
the code `new User()` can be moved to the `userInit()` method, this make the code more concise.